### ISSUE TITLE
[Keyboard Navigation - WIX - Setup Option] When user select OK button which available on the "Browse For Folder" popup window, then keyboard focus is not visible on the "Install location" editable field

### ISSUE THREAD

**Issue by sikoppar** [wixtoolset/issues#6041]
* Which version of WiX are you building with?
> 3.7.1224.1

* Which version of Visual Studio are you building with (if any)?
> Visual Studio 2017 (v141)

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?
>0.9.21.62588

* Which version of .NET are you building with?
> Not Applicable

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?
>Not Applicable

* Describe the problem and the steps to reproduce it.
User Experience:

If, the default keyboard focus does not land on "Install Location" editable field after selecting location , then the users who depends upon keyboard for navigation might get affected and does not know were the visual focus indicator is and unable to perform its function.

Pre Requist:
1. Download the installer from https://www.microsoft.com/en-us/download/details.aspx?id=54688
2. Navigate to NPS Extension for Azure MFA Setup and activate it by using Enter key.

Repro Steps:
1. Navigate to "Option" button of "NPS Extension for Azure MFA " popup window and select it.
2. Navigate all the controls of "Setup Option" popup window.
3. Verify if , Default keyboard focus lands on "Install Location" editable field after selecting location.

Actual Result :
When user select OK button which available on the "Browse For Folder" popup window, then keyboard focus is not visible on the "Install location" editable field. Keyboard focus only visible over the "Install location" editable field, when user press one extra tab.

Expected Result:
When user select OK button which available on the "Browse For Folder" popup window, then keyboard focus should be visible on the "Install location" editable field.

Note: When keyboard user press ESC key to close the "Browse For Folder" popup window, then keyboard focus does not land over the "Browse" button ( which is the origin control of the "Browse For Folder" popup window).

**Comment by BMurri** [comment]
That version of WiX is over 5 years old, and there have been changes to the WixStdBa since then. Do these issues replicate with v3.11.1?
